tests/unit/ext/auth/database.py

```diff
@@ -154,6 +155,22 @@ def use_fixtures(auth_fixture):
                                    'auth_context')
 
 
+@use_fixtures('admin_auth')
+class TestListUsers(DBAuthTestCase):
+    """ Test for list_users."""
+    def setUp(self):
+        from hil.ext.auth import database as dbauth
+        self.dbauth = dbauth
```

Comment:
Should probably just use the dbauth fixture from above.

Reply:
hmm, are you suggesting we should do:

```python
@use_fixtures("dbauth")
@use_fixtures("admin_auth")
def TestListUser(DBAuthTestCase):
    def test_list_users(self):
    result = json.loads(self.dbauth.list_users())
    
```
Since, `DBAuthTestCase` has already defined `setUp`? 

If true, should I clean up the other functions within this file along with this PR?

Reply:
> If true, should I clean up the other functions within this file along with this PR?

If it's just one or two places, then I am okay with it. Would prefer to have it's own PR though.

Reply:
Actually, yeah the way we're using fixtures in this module is rather inconsistent (I just happened to look at one of the *other* ways it's done in this module...). I think I'm okay with leaving this as is, and having a separate pr for cleaning this up.